### ISSUE TITLE
Transmission-QT 4.0.0-beta.1 doesn't start torrents added by magnet link.

### ISSUE THREAD

**Issue by blacklion** [transmission/transmission#4005]
### What is the issue?

When adding magnet link to 4.0.0-beta1 with "Start when added" option checked on, Transmission downloads metadata automatically but pauses data download after that and require manual "Start torrent" click to download data itself.

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

4.0.0-beta.1

**Comment by blind-oracle** [comment]
Same happens with `transmission-daemon` when adding torrents using magnets

**Comment by burlizzi** [comment]
I solved with this
```
diff --git a/libtransmission/torrent-magnet.cc b/libtransmission/torrent-magnet.cc
index 7d7db3941..3c4f84907 100644
--- a/libtransmission/torrent-magnet.cc
+++ b/libtransmission/torrent-magnet.cc
@@ -301,7 +301,7 @@ static void onHaveAllMetainfo(tr_torrent* tor, tr_incomplete_metadata* m)
     {
         delete tor->incompleteMetadata;
         tor->incompleteMetadata = nullptr;
-        tor->isStopping = true;
+        tor->isStopping = false;
         tor->magnetVerify = true;
         if (tor->session->shouldPauseAddedTorrents())
         {
```
so far it works but I am not confident enough that it will not create chaos and riots in the torrents pool
someone competent should check

**Comment by blind-oracle** [comment]
@burlizzi Thanks

I solved it by rolling back to 3.00 :) It leaks memory like crazy in Ubuntu 22 because of new OpenSSL, but still for now better than 4.0

**Comment by ckerr** [comment]
Closing as a duplicate of https://github.com/transmission/transmission/issues/3753
